installer/templates/phx_assets/webpack.config.js

```diff
@@ -45,9 +37,20 @@ module.exports = (env, options) => {
       ]
     },
     plugins: [
-      new MiniCssExtractPlugin({ filename: '../css/[name].css' }),
-      new CopyWebpackPlugin([{ from: 'static/', to: '../' }])
-    ]
-    .concat(devMode ? [new HardSourceWebpackPlugin()] : [])
+      new MiniCssExtractPlugin({ filename: '../css/app.css' }),
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: 'static/', to: '../' }
+        ]
+      })
+    ],
+    optimization: {
+      minimize: true,
```

Comment:
I believe this will force the assets to be minimised even in development, this may not be ideal for performance reasons if it's going to recompile on every change. Webpack appears to use the `mode` to automatically set this to `true` when using `--mode production` and `false` with `--mode development`.

Reference: [Mode documentation](https://webpack.js.org/configuration/mode/#mode-development)

Reply:
Was lazy here, but was afraid of the same. Totally agree.